src/Evolution/DiscontinuousGalerkin/Actions/VolumeTermsImpl.tpp

```diff
@@ -12,6 +12,7 @@
 #include "DataStructures/DataVector.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"
 #include "DataStructures/Variables.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
```

Comment:
@nilsdeppe I added this header since without this line I get compile error with the following message:
```
spectre/src/Evolution/DiscontinuousGalerkin/Actions/VolumeTermsImpl.tpp:232:13:
error: use of undeclared identifier 'determinant_and_inverse'
            determinant_and_inverse(logical_to_inertial_inverse_jacobian)
```
though I wonder why this has not been making any problems for other systems.

Reply:
Sure!

Reply:
Sure!